.github/workflows/ci.yml

```diff
@@ -157,6 +157,41 @@ jobs:
     - run: test ${{ steps.setup3.outputs.mpi }} == intelmpi
     - run: command -v impi_info
 
+  container:
+    runs-on: ubuntu-latest
+    container: ubuntu
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup MPICH
+      id:   setup1
+      uses: ./
+      with:
+        mpi: mpich
+    - run: test ${{ steps.setup1.outputs.mpi }} == mpich
+      shell: bash
```

Comment:
This `shell: bash` lines are not really needed. The command does not uses any bashisms, just the `test` command with three args.  Or am I missing something?

Reply:
```suggestion
```

Reply:
@dalcinl Actually it does look like a bashism since it won't work if I don't specify bash (where it presumably runs with `sh`). I can try writing the test a different way if you don't like the `shell: bash`.